docs/framework/database/schema-design.md

```diff
@@ -13,10 +13,10 @@ Another method of viewing CiviCRM's Database model is to use one of various MySQ
 1. Look at the list of tables in the database. The table name generally gives you a good idea of the type of data it stores.
 2. Pick one table and look closer. Let's look at `civicrm_country` (a relatively simple example) by opening the "Table inspector" for that table.
     - We can look at the list of columns in the table.
-    ![MySQL Workbench CiviCRM Country Columns](/img/mysql_workbench_civicrm_country_tables.png)
+    ![MySQL Workbench CiviCRM Country Columns](/docs/img/mysql_workbench_civicrm_country_tables.png)
```

Comment:
@toraritte I'm a little curious about this change. When you go to https://docs.civicrm.org/dev/en/latest/framework/database/schema-design/#mysql-workbench-workflow do you find that the "Country Columns" screenshot is not loading? Because I'm observing it loading (with the current path that begins with `/img`). I'm worried that adding `/docs` to the beginning of the path with cause the screenshot to no longer load. But if you're observing problems with the image loading, then I'd like to try and reproduce that on my end.